scripts/scripts-1.js

```diff
@@ -0,0 +1,171 @@
+function getMinNumberFromParams(numb_1, numb_2, numb_3, numb_4) {
+
+  const minimalNumber = Math.min(numb_1, numb_2, numb_3, numb_4);
+
+  if (numb_1 === undefined || numb_2 === undefined || numb_3 === undefined || numb_4 === undefined) {
+
+    return 'Undefined value!';
+
+  } else if (numb_1 === '' || numb_2 === '' || numb_3 === '' || numb_4 === '') {
+
+    return 'Your value is empty';
+
+  } else if (isNaN(minimalNumber)) {
+
+    return 'Your value is not a number!';
+
+  }
+
+  else
+
+    return minimalNumber;
+
+};
+
+function greatestСommonDivisor(numb_1, numb_2) {
```

Comment:
Here also not enough validation
<img width="566" alt="image" src="https://user-images.githubusercontent.com/84010501/191555523-12020b96-eb99-4827-91d7-496d7784e356.png">